### ISSUE TITLE
Is there an "inverse" of `.select(pl.first())`

### ISSUE THREAD

**Issue by cmdlineluser** [pola-rs/polars#8082]
### Research

- [X] I have searched the above polars tags on Stack Overflow for similar questions.

- [ ] I have asked my usage related question on [Stack Overflow](https://stackoverflow.com).

### Question about Polars

Question arises from reading #8046 

Is it possible to select all but the first column in an expression without knowing the name?
```
df = pl.Series(["a", "b", "c", "a", "a"]).to_frame()
```
`pl.first` can select the first: (there's also `pl.last`)
```
>>> df.to_dummies().select(pl.first())
shape: (5, 1)
┌─────┐
│ _a  │
│ --- │
│ u8  │
╞═════╡
│ 1   │
│ 0   │
│ 0   │
│ 1   │
│ 1   │
└─────┘
```
My initial thought was there may be a `pl.slice(1)` - `pl.slice` appears to be a module though.
```
# TypeError: 'module' object is not callable  

df.to_dummies().select(pl.slice(1))
```
```
>>> pl.slice
<module 'polars.slice' from '/usr/local/lib/python3.10/site-packages/polars/slice.py'>
```

**Comment by ritchie46** [comment]
The schema of a `DataFrame` and `LazyFrame` is known at all times, so you can chain with a `pipe` to select using the schema.

```python
df = pl.Series(["a", "b", "c", "a", "a"]).to_frame()
df.to_dummies().pipe(lambda df: df.select(df.columns[-1]))
```

**Comment by cmdlineluser** [comment]
Ah, `.pipe` is new to me, very nice.

Thanks @ritchie46 \o/

**Comment by cmdlineluser** [comment]
Just incase anybody finds this.

https://github.com/pola-rs/polars/pull/9456 has since been added.

```python
import polars.selectors as cs

df = pl.Series(["a", "b", "c", "a", "a"]).to_frame()

df.to_dummies().select(~cs.first())

# shape: (5, 2)
# ┌─────┬─────┐
# │ _b  ┆ _c  │
# │ --- ┆ --- │
# │ u8  ┆ u8  │
# ╞═════╪═════╡
# │ 0   ┆ 0   │
# │ 1   ┆ 0   │
# │ 0   ┆ 1   │
# │ 0   ┆ 0   │
# │ 0   ┆ 0   │
# └─────┴─────┘
 ```

**Comment by alexander-beedie** [comment]
> https://github.com/pola-rs/polars/pull/9456 has since been added.

Yup; that will be available in the upcoming 0.18.4 👍
